shoop/core/models/_orders.py

```diff
@@ -684,5 +684,42 @@ def can_edit(self):
 OrderLogEntry = define_log_model(Order)
 
 
+# note - @pikkupomo mentioned not to use a Polymorphic model due to migration troubles; hopefully multi-table
+# inheritance is better (seems to just be syntactic sugar around a one-to-one field with the parent)
+# One problem with defining the Purchase Order this way is upon PurchaseOrder creation - see _creator.py
+# another bummer is having to do Order.objects.exclude(purchaseorder__isnull=True) in order to get sales orders
+# although I suppose the latter can be fixed by adding a method to the query manager
+class PurchaseOrder(Order):
```

Comment:
I think you should implement the AbstractOrder. Leave the SalesOrder rename for later.